app/views/posts/show.html.erb

```diff
@@ -33,22 +33,21 @@
     </tbody>
   </table>
 </div>
-<div class="comments-posted">
-  <% if post.comments.exists? %>
-    <% post.comments.each do |c| %>
-      <%= "comment by #{c.user.name}: #{c.text}" %>
-      <br />
-    <% end %>
-  <% end %>
-</div>
-<%= form_for(new_comment, url: submissions_path, method: :post) do |f| %>
-  <%= f.text_area :text, cols: 60 %>
+<%= form_for(comment, url: post_comments_path(post.id), method: :post) do |f| %>
+  <%= f.text_area :text %>
   <br />
-  <%= f.hidden_field :post_id, :value => parent_id %>
+  <%= f.hidden_field :post_id, :value => post.id %>
```

Comment:
for consistency, convert that hash rocket (`=>`) into the newer Hash syntax (e.g. `value: post`)